src/rhsm/certificate.py

```diff
@@ -806,7 +766,6 @@ def branch(self, root):
 
     def _get_extensions_block(self, x509):
         """ Isolate the block of text with the extensions. """
-        text = x509.as_text()
         p = Popen(['openssl', 'x509', '-text', '-certopt', 'ext_parse'],
```

Comment:
Oh man...  :sob: Maybe we can get rid of this weird shelling out to grab the extensions while were in here?